spec/system/visitor_entry/manager_see_visitor_entries_spec.rb

```diff
@@ -16,19 +16,32 @@
 
   it 'and the list of visitors appears in descending order of entry (created_at)' do
     manager = create :manager
-    condo = create :condo
-    create :visitor_entry, condo:, full_name: 'Nome Primeiro Visitante'
-    create :visitor_entry, condo:, full_name: 'Nome Último Visitante'
+    first_condo = create :condo
+    second_condo = create :condo
+    create :visitor_entry, condo: first_condo, full_name: 'Nome Primeiro Visitante'
+    create :visitor_entry, condo: first_condo, full_name: 'Nome Segundo Visitante'
+    create :visitor_entry, condo: second_condo, full_name: 'Nome Terceiro Visitante'
 
     login_as manager, scope: :manager
-    visit condo_visitor_entries_path condo
+    visit condo_visitor_entries_path first_condo
 
     within('.table > tbody > tr:nth-child(1)') do
-      expect(page).to have_content 'Nome Último Visitante'
+      expect(page).to have_content 'Nome Segundo Visitante'
     end
     within('.table > tbody > tr:nth-child(2)') do
       expect(page).to have_content 'Nome Primeiro Visitante'
     end
+    expect(page).not_to have_content 'Nome Terceiro Visitante'
+  end
+
+  it 'and sees if list is empty' do
```

Comment:
Mesma coisa que acima, o `it` fica estranho aqui quando combinado com o `describe`.